python/lsst/sims/catUtils/baseCatalogModels/OpSim3_61DBObject.py

```diff
@@ -107,15 +104,25 @@ def executeConstrainedQuery(self, spatialBound, constraint=None):
         results = self.execute_arbitrary(query, dtype=dtype)
         return results
 
-    def getObservationMetaData(self, obshistid, radiusDeg, makeCircBounds=True, makeBoxBounds=False):
+    def getObservationMetaData(self, searchCenter, searchRadius, constraint=None,
+                               fovRadius=1.75, makeCircBounds=True, makeBoxBounds=False):
         """
-        This class creates an ObservationMetaData object suitable for generating a PhoSim input
-        catalog based on an actual OpSim pointing.
+        This method will perform a query on the OpSim3_61 for pointings within a given region
```

Comment:
Can we provide a url here in the documentation to where the list of opsim columns can be found? I think that might be useful.